### ISSUE TITLE
[COGNITION]: VAOS Past Appointments - Focus SHOULD be set on the updated label when results are returned

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10888]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Focus is being returned to the Past Appointments H2 when users press the Update button. This seemed like the right approach when we were improving this past appointments workflow, but now that I see it in action, it would be better to set focus on the label of appointments being shown. This improvement will give screen reader users a better understanding that their search returned meaningful data. Screen shot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* [ ] Focus is being set on the correct text block
* [ ] No yellow focus halo is shown
* [ ] Screen readers announce the updated past appointments correctly

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/past

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2020-07-02 at 3 46 24 PM](https://user-images.githubusercontent.com/934879/86408256-4488bd00-bc7c-11ea-89a9-d19586162ea2.png)

**Comment by 1Copenut** [comment]
@jbalboni The behavior is spot-on when users press the Update button on Past Appointments. Can we keep focus on the Past Appointments tab when users make it the selected tab? 

When users open the Past Appointments tab, we don't want to skip past the form controls; screen reader users might not understand there was a way to change their view. I'm attaching an animated GIF showing this behavior.

---
![past-appointments-tab-load](https://user-images.githubusercontent.com/934879/87178660-1b32e700-c2a3-11ea-99d2-5c2af3b3deb5.gif)

**Comment by vbahinwillit** [comment]
Per Slack conversation with Trevor:
_"I am going to reverse my previous understanding and say we should keep focus on the tab ( Past appointments ) when data is loaded the first time. We will only move focus to the “Showing appointments…” when users press the Update button when using the filters dropdown."_

**Comment by jbalboni** [comment]
@1Copenut This should be ready to test on staging.

**Comment by 1Copenut** [comment]
I've confirmed this with VoiceOver and JAWS. Finishing up with NVDA now. Things are looking good. Moving to close.
